### PR TITLE
12582 mhr callback url

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12582

*Description of changes:*
add callbackURL for submitSelectedMhr

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Review by dimak1 — APPROVED** [review]

File: ppr-ui/src/utils/mhr-api-helper.ts [hunk 1]
```diff
@@ -122,6 +123,16 @@ export async function submitSelectedMhr (
     }
   }
 
+  if (shouldCallback || selected.length >= 75) {
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/mhr-api-helper.ts]: If we are putting the logic here to enforce the callback based on arr length, do we need the argument to do it manually?

**Review by cameron-eyds — APPROVED** [review]
Looks good! 
I'm not against an override argument although if you don't need it now, typically would add it in when you do need it :) 
Less is more right! haha

File: ppr-ui/src/utils/mhr-api-helper.ts (hunk 1 above)
Comment (garyfan1) [reply ppr-ui/src/utils/mhr-api-helper.ts]: I leave it there because callback is optional for sizes that are less than 75?

**Comment by garyfan1** [comment]
oh I see your point; I don't need to add things for the future. I can definitely remove those if we really don't want to keep it.

**Comment by cameron-eyds** [comment]
> oh I see your point; I don't need to add things for the future. I can definitely remove those if we really don't want to keep it.

For sure, i would only add if we needed it now, thanks 👍 
If you were building a shared component that was going to accessible for many implementations, it would make sense to include options up front.

**Review by cameron-eyds — APPROVED** [review]

**Pull request merged**